api/src/main/java/org/openmrs/util/VelocityExceptionHandler.java

```diff
@@ -43,4 +44,10 @@ public Object methodException(Class claz, String method, Exception e) throws Exc
 		throw e;
 	}
 	
+	@Override
+	public Object methodException(Context context, Class claz, String method, Exception e, Info info) {
+		// TODO Auto-generated method stub
```

Comment:
What is the use of the above comment?

Reply:
sorry @dkayiwa  for responding this LATE, have just seen it now.
the above comment "TODO Auto-generated method stub" was just an auto generated comment

Reply:
@gitcliff,  auto generated from where? In any case, it is not advisable to submit code containing any TODO. So why have a duplicate method `methodException(Context context, Class claz, String method, Exception e, Info info)` for the changes made in this class?

Reply:
Also checkout this https://wiki.openmrs.org/display/docs/Java+Conventions .

Reply:
okay @Ruhanga thanks for this,,,going to check it out